src/db.c

```diff
@@ -1280,6 +1280,9 @@ int db_col_attr_get(const struct db_filter_col *col,
 	case SCMP_FLTATR_CTL_LOG:
 		*value = col->attr.log_enable;
 		break;
+	case SCMP_FLTATR_SPEC_ALLOW:
+		*value = col->attr.spec_allow;
+		break;
```

Comment:
It would be good to add a check for this attribute to `tests/13-basic-attrs.{c,py}`.

Reply:
You added the C test - that's great - but forgot the Python test unfortunately.